.github/workflows/merge-dependabot.yml

```
@@ -0,0 +1,29 @@
+name: Dependabot auto-approve
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
```

```suggestion
    if: github.actor == 'plasmapy-requirements-bot'
```

🔧 I forgot to mention in #2647 that the automated requirements updates are done through our own custom `plasmapy-requirements-bot` rather than dependabot.  I'm not sure if we'd need to include the `[bot]` at the end here.

---

Ah okay. Both dependabot and the plasmapy-requrirements-bot have the "app/" prefix when I add them as a filter to the PR search under "author", so I'm going to guess we need to add the [bot] suffix for the plasmapy bot also.